jep/1/README.adoc

```diff
@@ -0,0 +1,966 @@
+help write, implement, discussion, or offer feedback about the JEP.
+One contributor might do only one or any combination of these
+of these things during any part of the life of a JEP.
+
+Because the JEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal footnoteref:[repo, https://github.com/jenkinsci/jep].
+
+[NOTE]
+====
+For non-text files added to the repository, such as images, the files must be
+readable with free and open source tools, and should ideally be editable by
+other contributors. For example, use SVG rather than PSD for vector graphics.
+====
+
+
+=== JEP Types
+
+There are three kinds of JEP:
+
+. A **Standards Track** JEP describes a new feature or implementation
+ for Jenkins. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of Jenkins, moving forward.
+. An **Informational** JEP describes a Jenkins design issue, or
+ provides general guidelines or information to the Jenkins community,
+ but does not propose a new feature. Informational JEPs do not
+ necessarily represent a Jenkins community consensus or
+ recommendation, so users and implementers are free to ignore
+ Informational JEPs or follow their advice.
+. A **Process** JEP describes a process surrounding Jenkins, or
+ proposes a change to (or an event in) a process. Process JEPs are like
+ Standards Track JEPs but apply to areas other than the Jenkins codebase
+ itself. They may propose an implementation, but not for what would be
+ generally considered the Jenkins codebase; they often require community
+ consensus; unlike Informational JEPs, they are more than recommendations, and
+ users are typically not free to ignore them. Examples include procedures,
+ guidelines, changes to the decision-making process, and changes to the tools
+ or environment used in Jenkins development. Any meta-JEP (JEP focusing on the
+ improvement of the Jenkins Enhancement Proposal process) is also considered a
+ Process JEP.
+
+=== JEP Terminology
+
+==== BDFL
+
+There are several references in this JEP to the "BDFL". This acronym stands for
+"link:https://en.wikipedia.org/wiki/Benevolent_dictator_for_life[Benevolent Dictator for Life]".
+The BDFL is a single person whose role in the project is to act as decision maker
+when one is needed. The responsibilities of the BDFL are:
+
+* Review JEPs and decide whether they will will be accepted
+* Delegate JEP Reviewer responsibilities to other contributors as appropriate
+* Resolve disputes or arguments that cannot be resolved by consensus
+* Take any other actions as part of the JEP process that they deem best of the community
+* Clearly communicate the reasoning for any actions taken or decisions made
+* Refrain from using their power, letting the community self-govern whenever possible
+
+The BDFL may override the decision any other contibutors.
```

Comment:
As we discussed with @kohsuke, these sentences need to be removed or adjusted. My IMHO: There should be a way to override the BDFL's decision by the Governance meeting at least.